lib/build.api.js

```diff
@@ -38,18 +39,52 @@ module.exports = function (Aquifer) {
         return;
       }
 
-      // Run drush make.
-      drush.init().then(function() {
-        drush.exec('make ' + make + ' ' + self.destination)
-        .then(function(res) {
-          fs.symlinkSync(self.project.absolutePaths.settings, path.join(self.destination, 'sites/default/site.settings.php'), 'file');
-          //fs.symlinkSync(path.join(self.destination, 'sites/all/themes/contrib'), self.project.absolutePaths.themes.contrib, 'directory');
-          //fs.symlinkSync(path.join(self.destination, 'sites/all/modules/contrib'), self.project.absolutePaths.modules.contrib, 'directory');
-          //fs.symlinkSync(path.join(self.destination, 'sites/all/modules/custom'), self.project.absolutePaths.modules.contrib, 'directory');
-          //fs.symlinkSync(path.join(self.destination, 'sites/all/modules/features'), self.project.absolutePaths.modules.features, 'directory');
+      // Initialize drush and the build promise chain.
+      drush.init()
+        // Delete current build.
+        .then(function () {
+          var def = new Deferred();
+
+          fs.remove(self.destination, function (err) {
```

Comment:
Can we just use fs.removeSync? We're deferring this but we don't need to do that if we just use the sync version of this function.